src/model.py

```diff
@@ -220,6 +228,23 @@ def create_response(self, vllm_output):
         )
         return pb_utils.InferenceResponse(output_tensors=[triton_output_tensor])
 
+    def create_stream_response(self, vllm_output, last_outputs):
+        """
+        Parses the output from the vLLM engine, extracts only newly generated
+        text and packs it into Triton response.
+        """
+        if last_outputs is None:
+            return self.create_response(vllm_output, exclude_input_in_output=True)
+
+        text_outputs = [
+            (output.text[len(last_output.text) :]).encode("utf-8")
+            for output, last_output in zip(vllm_output.outputs, last_outputs.outputs)
```

Comment:
it occurs to me - you could store the current length of the previous output instead of the output itself - if its not needed elsewhere